### DISCUSSION TITLE
Import bottle or create bottle in external drive

### DISCUSSION THREAD

**Discussion by lossiro** [Whisky-App/Whisky#290] [Q&A]
Hello everyone,

After updating to the latest version 1.1.0, I can't import or create bottles on my external drive. However, if I install the previous version 1.0.1, everything works fine. I can import and create bottles on my external drive without any problems.

I'm not sure what's causing the issue, but I wanted to report it in case anyone else is experiencing the same problem.

I'm using Macbook M1 Pro

**Comment by titovskiy** [comment]
have same issue =/ could not use my external ssd (usb-c m2) for install apps
bottle created normally, but if I try to install any app — no respond

**Comment by QWERTYLIN123** [comment]
Had the exact same problem with my M2 when trying to create a bottle to my external SSD. I just updated Whisky to 1.1.1 and that fixed the problem.
